specs/_features/eip7594/polynomial-commitments-sampling.md

```diff
@@ -20,13 +20,13 @@
   - [FFTs](#ffts)
     - [`_fft_field`](#_fft_field)
     - [`fft_field`](#fft_field)
+    - [`coset_fft_field`](#coset_fft_field)
```

Comment:
Theres a test_fft method, so blocking this until we add a similar method to test `coset_fft`

Reply:
unblocked with [92d3fcb](https://github.com/ethereum/consensus-specs/pull/3781/commits/92d3fcbc4a4336e6837cc4cc45cffb5b62fee0d9)